packages/common/src/hexToResourceId.test.ts

```diff
@@ -0,0 +1,11 @@
+import { describe, it, expect } from "vitest";
+import { hexToResourceId } from "./hexToResourceId";
+
+describe("hexToResourceId", () => {
+  it("can convert from hex string", () => {
+    const resourceId = hexToResourceId("0x6e616d65737061636500000000006e616d650000000000000000000000007462");
+    expect(resourceId.namespace).toMatchInlineSnapshot('"namespace"');
+    expect(resourceId.name).toMatchInlineSnapshot('"name"');
+    expect(resourceId.type).toMatchInlineSnapshot('"table"');
```

Comment:
ooc what's the difference between `toMatchInlineSnapshot` and `toEqual`?

Reply:
inline snapshots are serialized and will give you when running tests to update the snapshot

a bit less of a hassle than having a hardcoded value in `toEqual` that you have to manually update